h/session.py

```diff
@@ -10,6 +11,7 @@ def model(request):
     session['csrf'] = request.session.get_csrf_token()
     session['userid'] = request.authenticated_userid
     session['groups'] = _current_groups(request)
+    session['features'] = features.features_status(request)
```

Comment:
I'd rather we didn't use view functions as helper functions. Move `return {k: flag_enabled(request, k) for k in FEATURES.keys()}` into a helper function called something like `all` and then have both the view function in `h.features` and this call `features.all(request)`.